app/src/main/java/org/simple/clinic/storage/Migration_50_51.kt

```diff
@@ -0,0 +1,13 @@
+package org.simple.clinic.storage
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Suppress("ClassName")
+class Migration_50_51 : Migration(50, 51) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.execSQL("""ALTER TABLE "Encounter" ADD COLUMN "syncStatus" TEXT NOT NULL DEFAULT "DONE" """)
```

Comment:
Curious, why are we setting this column value to `DONE`. How do we know that all the encounters have been synced? Context please.

Reply:
We do not have `Encounter` syncing implemented yet. Hence when this migration runs, the app won't have any `Encounter` records. So it's safe to assume this status.

The reason I am not adding `PENDING` as a default status is because that should only be added when you want records to be synced.
The other status is `INVALID` which will not be appropriate either because it is an error status.